### ISSUE TITLE
Latency over time - median value displayed for p95 and p99

### ISSUE THREAD

**Issue by foliv57** [artilleryio/report-viewer#7]
Hi,

I'm using the latest version of report viewer and, for some reason, in the "Latency over time" graphic, the p95 and p99 lines are both displaying the median value.

![image](https://user-images.githubusercontent.com/59969260/155187181-110366dc-df38-4106-b4cc-b5137bd983bb.png)

![image](https://user-images.githubusercontent.com/59969260/155188088-4d1b4910-a407-4594-87c9-ca9b63d293a2.png)

Attached the JSON report.

Thank you
[report.zip](https://github.com/artilleryio/report-viewer/files/8118928/report.zip)

**Comment by cfryerdev** [comment]
Yeah, looks like a bad paste in `graph_latency.jsx` on lines `53` and `60`.

**Comment by cfryerdev** [comment]
I have a PR open for this fix, waiting for an approval.

**Comment by cfryerdev** [comment]
@foliv57 can you verify now that you are seeing the correct values? After which we can close this issue, thanks a ton for letting me know!

**Comment by foliv57** [comment]
Hi @cfryerdev ,

I confirm that the change fixed the issue.
![image](https://user-images.githubusercontent.com/59969260/164178445-ffb91be5-c3eb-4e39-8ed4-2b367053b0a1.png)

Thank you for this fix

**Comment by cfryerdev** [comment]
Excellent. @hassy can close this bug.
